src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+                    throw new EmptyInputException("Empty input! Try again (o|o)\n");
+                } else if (sentence.equals("bye")) {
+                    Chatter.bye();
+                    break;
+                } else if (sentence.equals("list")) {
+                    Chatter.list();
+                } else if (sentence.startsWith("done")) {
+                    Chatter.done(sentence);
+                } else if (sentence.startsWith("todo")) {
+                    Task newTask = Chatter.taskBank.addTodo(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("deadline")) {
+                    Task newTask = Chatter.taskBank.addDeadline(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("event")) {
+                    Task newTask = Chatter.taskBank.addEvent(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("delete")) {
+                    Chatter.deleteTask(sentence);
+                } else if (sentence.startsWith("clear")) {
+                    Chatter.clear();
+                } else {
+                    throw new IrregularInputException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(" +
+                            "\nPlease try again!");
+                }
+            } catch (IrregularInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (RepeatedCompletionException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (EmptyInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (NumberFormatException e) {
+                printDashLine();
+                System.out.printf("Not a valid number. Try again%n");
+                printDashLine();
+            }
+        }
+    }
+
+    static void add(Task newTask) {
+        printDashLine();
+        System.out.printf("Got it. I've added this task: %n " +
+                newTask +
+                "%nNow you have " + taskBank.getTaskSize() +
+                " tasks in the list.%n");
+        printDashLine();
+        taskBank.exportTasks();
+    }
+
+    static void bye() {
+        printDashLine();
+        System.out.printf("Bye. Hope to see you again soon!%n");
+        printDashLine();
+    }
+
+    static void list() {
+        printDashLine();
+        System.out.printf("Here are the tasks in your list:%n");
+        taskBank.printList();
+        printDashLine();
+    }
+
+    static void done(String sentence) throws RepeatedCompletionException, NumberFormatException {
```

Comment:
Consider using a more meaningful method name so that other programmers and you can refer to your code easily. For example, you could name this method as `markAsDone`. This applies to other methods as well.